caffe2/experiments/python/SparseTransformer.py

```diff
@@ -106,13 +106,13 @@ def transFCRelu(cur, id2node, name2id, ops, model):
         cur.visited = True
         pre = cur
         flag = False
-        for _, temp in cur.ops.iteritems():
+        for _, temp in cur.ops.items():
```

Comment:
Using `six.iteritems()` would maintain the performance benefit in Py2. Not sure if the maintainers would want to add a new dependency, though.